apps/central-scan/frontend/src/screens/unconfigured_election_screen_wrapper.tsx

```diff
@@ -3,26 +3,34 @@ import {
   Main,
   Screen,
   UnconfiguredElectionScreen,
-  UsbDriveStatus,
-  useExternalStateChangeListener,
+  useQueryChangeListener,
 } from '@votingworks/ui';
+import { assert } from '@votingworks/basics';
 import { MainNav } from '../components/main_nav';
-import { configureFromBallotPackageOnUsbDrive, logOut } from '../api';
+import {
+  configureFromBallotPackageOnUsbDrive,
+  getUsbDriveStatus,
+  legacyUsbDriveStatus,
+  logOut,
+} from '../api';
 
 interface Props {
   isElectionManagerAuth: boolean;
-  usbDriveStatus: UsbDriveStatus;
 }
 
 export function UnconfiguredElectionScreenWrapper({
   isElectionManagerAuth,
-  usbDriveStatus,
 }: Props): JSX.Element {
+  const usbDriveStatusQuery = getUsbDriveStatus.useQuery();
+  // USB drive status is guaranteed to exist because app root will not render
+  // this component until the USB drive query succeeds.
+  assert(usbDriveStatusQuery.isSuccess);
+
   const logOutMutation = logOut.useMutation();
   const configureMutation = configureFromBallotPackageOnUsbDrive.useMutation();
 
-  useExternalStateChangeListener(usbDriveStatus, (newUsbDriveStatus) => {
-    if (newUsbDriveStatus === 'mounted') {
+  useQueryChangeListener(usbDriveStatusQuery, (newUsbDriveStatus) => {
```

Comment:
Can this move to the backend now? Or do you want to save that for later?

Reply:
It didn't occur to me to move it to the backend. Are you suggesting that, instead of triggering the configuration from the frontend, it's triggered directly from the backend? Or are you suggesting something else?

If we were to do that, then we'd need a way to get the configuration status / error to the frontend. I am thinking we might augment the the `getUsbDriveStatus` API endpoint to have extra data about configuration status / attempt(s).

Reply:
Yes, that's what I'm suggesting. Why should the frontend be involved at all? The user doesn't need to interact with the screen during configuration.

I think it would make the most sense to put error/in-progress status in endpoint gets the current configuration (currently, `getElectionDefinition` is the closest thing)

Reply:
> I am thinking we might augment the the getUsbDriveStatus API endpoint to have extra data about configuration status / attempt(s).

I did something like this while working on the CVR resync modal!

https://github.com/votingworks/vxsuite/blob/021f5ea2247967b5d5ee3c2994537b0643d46039/apps/scan/backend/src/app.ts#L106-L118